gomint-server/src/main/java/io/gomint/server/world/generator/vanilla/VanillaGeneratorImpl.java

```diff
@@ -284,6 +288,17 @@ public VanillaGeneratorImpl(World world, GeneratorContext context) {
         }
     }
 
+    private int getRandomPort() {
+        return IntStream.range(50000, 60000).filter(port -> {
+            try(DatagramSocket datagramSocket = new DatagramSocket()) {
+                datagramSocket.bind(new InetSocketAddress("127.0.0.1", port));
+                return true;
+            } catch(SocketException e) {
+                return false;
+            }
+        }).findFirst().orElseThrow();
```

Comment:
Let the system find a free port:
```suggestion
        try(DatagramSocket datagramSocket = new DatagramSocket(0)) {
            return datagramSocket.getLocalPort()
        } catch(SocketException ex) {
            throw new RuntimeException("Could not open socket to find next free port", ex);
        }
```